subsys/bluetooth/host/gatt.c

```diff
@@ -1048,6 +1048,11 @@ static int bt_gatt_store_cf(uint8_t id, const bt_addr_le_t *peer)
 
 }
 
+static inline bool is_host_managed_ccc(const struct bt_gatt_attr *attr)
```

Comment:
I'm pretty sure the `inline` is completely unnecessary (i.e. premature optimization). Only add those if for some reason the compiler doesn't do the right thing.